source/linux/Foundational_Components/Power_Management/pm_wakeup_sources.rst

```diff
@@ -109,66 +143,130 @@ For example, if you wish to wakeup from Deep Sleep or MCU Only mode in 10 second
+        [   34.387141] omap8250 2800000.serial: PM domain pd:146 will not be powered off
+        [   34.387736] ti-sci 44043000.system-controller: ti_sci_cmd_set_device_constraint: device: 117: state: 1: ret 0
+        [   34.416958] remoteproc remoteproc0: stopped remote processor 79000000.r5f
+        [   34.420565] Disabling non-boot CPUs ...
+        [   34.422781] psci: CPU1 killed (polled 0 ms)
+        [   34.426363] psci: CPU2 killed (polled 0 ms)
+        [   34.429526] psci: CPU3 killed (polled 0 ms)
+        [   34.430459] Enabling non-boot CPUs ...
+        [   34.430798] Detected VIPT I-cache on CPU1
+        [   34.430841] GICv3: CPU1: found redistributor 1 region 0:0x00000000018a0000
+        [   34.430895] CPU1: Booted secondary processor 0x0000000001 [0x410fd034]
+        [   34.431987] CPU1 is up
+        [   34.432232] Detected VIPT I-cache on CPU2
+        [   34.432262] GICv3: CPU2: found redistributor 2 region 0:0x00000000018c0000
+        [   34.432306] CPU2: Booted secondary processor 0x0000000002 [0x410fd034]
+        [   34.433233] CPU2 is up
+        [   34.433485] Detected VIPT I-cache on CPU3
+        [   34.433515] GICv3: CPU3: found redistributor 3 region 0:0x00000000018e0000
+        [   34.433557] CPU3: Booted secondary processor 0x0000000003 [0x410fd034]
+        [   34.434504] CPU3 is up
+        [   34.434948] ti-sci 44043000.system-controller: ti_sci_resume: wakeup source: 0x50
+        [   34.447824] am65-cpsw-nuss 8000000.ethernet: set new flow-id-base 19
+        [   34.463954] am65-cpsw-nuss 8000000.ethernet eth0: PHY [8000f00.mdio:00] driver [TI DP83867] (irq=POLL)
+        [   34.463980] am65-cpsw-nuss 8000000.ethernet eth0: configuring for phy/rgmii-rxid link mode
+        [   34.477401] am65-cpsw-nuss 8000000.ethernet eth1: PHY [8000f00.mdio:01] driver [TI DP83867] (irq=POLL)
+        [   34.477414] am65-cpsw-nuss 8000000.ethernet eth1: configuring for phy/rgmii-rxid link mode
+        [   34.661705] OOM killer enabled.
+        [   34.664848] Restarting tasks ... done.
+        [   34.670624] random: crng reseeded on system resumption
+        [   34.676468] platform 79000000.r5f: Core is off in resume
+        [   34.681906] remoteproc remoteproc0: powering up 79000000.r5f
+        [   34.687692] remoteproc remoteproc0: Booting fw image am62p-mcu-r5f0_0-fw, size 58344
+        [   34.699283] rproc-virtio rproc-virtio.5.auto: assigned reserved memory node mcu-r5fss-dma-memory-region@9b800000
+        [   34.710642] virtio_rpmsg_bus virtio0: rpmsg host is online
+        [   34.716279] virtio_rpmsg_bus virtio0: creating channel ti.ipc4.ping-pong addr 0xd
+        [   34.717435] rproc-virtio rproc-virtio.5.auto: registered virtio0 (type 7)
+        [   34.724381] virtio_rpmsg_bus virtio0: creating channel rpmsg_chrdev addr 0xe
+        [   34.731147] remoteproc remoteproc0: remote processor 79000000.r5f is now up
+        [   34.754176] PM: suspend exit
 
     .. note::
```

Comment:
The indentation of directives like code-block, note, etc doesn't follow the guidelines added in CONTRIBUTING - https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#indentation-and-whitespace

Sections should clearly line up with directive

Reply:
Done